tests/Functional/WithMigrationTest.php

```diff
@@ -28,22 +28,19 @@ final class WithMigrationTest extends KernelTestCase
     /**
      * @test
      */
-    public function it_generates_a_valid_schema(): void
+    public function it_can_use_schema_reset_with_migration(): void
```

Comment:
I don't know why, but in some case, when this test class used to have 2 tests, the second one was failing. Hence why I merged them